docs/usage.rst

```diff
@@ -0,0 +1,7 @@
+=====
+Usage
+=====
+
+To use SpyQL in a project::
+
+    import spyql
```

Comment:
Since we don't have a well-defined API for programmatically interacting with spyql shouldn't this file include the CLI examples we currently have in the readme?

Reply:
agree, we should remove that file, it is not being included in the docs... the docs still need some improvements